collectors/python.d.plugin/example/metadata.yaml

```diff
@@ -0,0 +1,136 @@
+      description: ''
+setup:
+  prerequisites:
+    list: []
+  configuration:
+    file:
+      name: python.d/example.conf
+      description: ''
+    options:
+      description: |
+        There are 2 sections:
+
+        * Global variables
+        * One or more JOBS that can define multiple different instances to monitor.
+
+        The following options can be defined globally: priority, penalty, autodetection_retry, update_every, but can also be defined per JOB to override the global values.
+
+        Additionally, the following table contains all the options that can be configured inside a JOB definition.
+
+        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
+      folding:
+        title: Config options
+        enabled: true
+      list:
+        - name: num_lines
+          description: The number of lines to create.
+          default_value: 4
+          required: false
+        - name: lower
+          description: The lower bound of numbers to randomly sample from.
+          default_value: 0
+          required: false
+        - name: upper
+          description: The upper bound of numbers to randomly sample from.
+          default_value: 100
+          required: false
+        - name: update_every
+          description: Sets the default data collection frequency.
+          default_value: 1
+          required: false
+        - name: priority
+          description: Controls the order of charts at the netdata dashboard.
+          default_value: 60000
+          required: false
+        - name: autodetection_retry
+          description: Sets the job re-check interval in seconds.
+          default_value: 0
+          required: false
+        - name: penalty
+          description: Indicates whether to apply penalty to update_every in case of failures.
+          default_value: yes
+          required: false
+        - name: name
+          description: Job name. This value will overwrite the `job_name` value. JOBS with the same name are mutually exclusive. Only one of them will be allowed running at any time. This allows autodetection to try several alternatives and pick the one that works.
+          default_value: ''
+          required: false
+    examples:
+      folding:
+        enabled: true
+        title: Config
+      list:
+        - name: Basic
+          folding:
+            enabled: false
+          description: A basic example configuration.
+          config: |
+            four_lines:
+                name: "Four Lines"
+                update_every: 1
+                priority: 60000
+                penalty: yes
+                autodetection_retry: 0
+                num_lines: 4
+                lower: 0
+                upper: 100
+troubleshooting:
+  problems:
+    list: []
+alerts: []
+metrics:
+  folding:
+    title: Metrics
+    enabled: false
+  description: ""
+  availability: []
+  scopes:
+  - name: global
```

Comment:
can you format this file in vscode? Some identation is messed up (from its generation), so yamllint complains

Reply:
Will do

Reply:
whats the problem here - am not sure

Reply:
im not sure what to change here